Tables/FE8 Nightmare modules/Class & Character editors/FE8 Character Editor.csv

```diff
@@ -136,8 +136,8 @@ Grado Soldier 6,0x24d,0x2a9,0x85,0x5,0x0,0x3,0,0x0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0
 Grado Soldier 7,0x24d,0x2a9,0x86,0x13,0x0,0x3,0,0x0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,60,20,30,35,15,15,25,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0
 Grado Soldier 8,0x24d,0x2a9,0x87,0xf,0x0,0x3,0,0x0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,60,20,30,35,15,15,25,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0
 Grado Soldier 9,0x24d,0x2a9,0x88,0xd,0x0,0x3,0,0x0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,60,20,30,35,15,15,25,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0
-Chapter 5 Generic,0x24d,0x2a9,0x89,0x25,0x0,0x3,0,0x0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,60,20,30,35,15,15,25,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0
-Grado Soldier 11 (Duessel's),0x24d,0x2a9,0x8a,0x44,0x0,0x3,0,0x0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,60,20,30,35,15,15,25,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0
+Grado Soldier 10 (Duessel's),0x24d,0x2a9,0x89,0x25,0x0,0x3,0,0x0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,60,20,30,35,15,15,25,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0
+Chapter 5 Generic,0x24d,0x2a9,0x8a,0x44,0x0,0x3,0,0x0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,60,20,30,35,15,15,25,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0
```

Comment:
Were you actually going to change the stats?
Per the diff:
-Ch5Generic,0x23d,0x298,0x4b,0x2d,0x36,0x0,1,0x0,1,2,1,0,2,1,3,1,0,0,0,0,0,0,0,0,0,85,50,10,10,10,15,30,0x0,0x0,0x0,0x0,0x0,0x0,0x80,0x0,0x0,0x0,0x0,0x0,0x0,0x0